apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx

```diff
@@ -863,68 +895,86 @@ export const AttributeSelectionOIDC: FunctionComponent<AttributeSelectionOIDCPro
-                                                            .get("LOCAL_CLAIMS")
-                                                    );
-                                                } }
-                                            > Attribute.
-                                            </Link>
-                                        </Trans>
-                                    </Hint>
-                                )
-                            }
+                                    <Link
+                                        external={ false }
+                                        onClick={ () => {
+                                            history.push(
+                                                AppConstants.getPaths()
+                                                    .get("ATTRIBUTE_MAPPINGS")
+                                                    .replace(":type", ClaimManagementConstants.OIDC)
+                                            );
+                                        } }
+                                    > Attributes.
+                                    </Link>
+                                </Trans>
+                            </Hint>
+                        </Grid.Column>
+                    </Grid.Row>
+                    <Grid.Row className="mt-5">
+                        <Grid.Column>
+                            <Grid.Row>
+                                <Form>
+                                    <Form.Field>
+                                        <div className="display-flex">
+                                            <CopyInputField
+                                                className="copy-input spaced"
+                                                value={ selectedScopes.join(" ") }
+                                                data-componentid={ `${ componentId }-selected-scope-area` }
+                                            />
+                                        </div>
+                                    </Form.Field>
+                                </Form>
+                            </Grid.Row>
                         </Grid.Column>
                     </Grid.Row>
+                    <Hint>
+                        <Trans
+                            i18nKey={
+                                "console:develop.features.applications.edit.sections.attributes." +
+                                                "selection.selectedScopesComponentHint"
+                            }
+                        >
+                            Request these scopes from your application to retrieve the user attributes 
+                            you selected.
+                        </Trans>
+                        {
+                            getLink("develop.connections.editApplication.attributeManagement" +
+                            ".manageOIDCScopes") !== undefined
```

Comment:
We can use `&&` instead of checking if it is not `undefined`.

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3640/commits/36d8f442d1333ae449424157d70d879c508e5d43